intro/notebooks-intro/notebooks-intro.ipynb

```diff
@@ -0,0 +1,311 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "1. Edit the following markdown cell with the *CHANGE_ME* text.\n",
+    "To do this, click the cell text and press `Enter`.\n",
+    "Change the text to *CHANGED*, and then press `Ctrl+Enter` to render the cell.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "CHANGE_ME"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "2. Edit the following code cell to print the message *Hello Red Hat*.\n",
+    "Press `Ctrl+Enter` to render the result."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "print(\"Hello!\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "3. Execute the following cell several times to verify that it returns the same *Process Identifier* (PID) number.\n",
+    "This is the PID of the Python kernel. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.getpid()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "4. Use a cell to assign the `hey` string to a variable called `test`, and print the value in a different cell.\n",
+    "Verify that the cell which prints the value renders the string\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "test = \"hey\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "print(test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "5. Display the `redhat.png` image in the notebook's directory by using markdown.\n",
+    "To do this, replace the `CHANGE_ME` string in the image name.\n",
+    "\n",
+    "![Red Hat logo](CHANGE_ME.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "6. Display an image by using Python code.\n",
+    "For this, replace the `CHANGE_ME` string with the `redhat.png` file name."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from IPython import display\n",
+    "\n",
+    "display.Image(\"CHANGE_ME\", width=600, height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "7. Uncomment the last two lines from the following source code and run the cell to plot a sine graph.\n"
```

Comment:
Excuse my ignorance on this one, I just want to check that you meant to say "sine" graph and not "line" graph. I don't know what a sine graph is and it has close spelling to line, just want to check it is not a typo.

Reply:
No worries Sam. A sine is a math function that prints a graph like the following.
https://images.app.goo.gl/6Nhi19y9GZ9t6fb2A